src/components/BarList/BarLabel.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, ReactNode } from 'react'
+import { composeClasses } from 'lib/classes'
+import { Size } from '../../interfaces/types'
+import Text from '../Typography'
+import Anchor from '../Anchor/Anchor'
+
+export interface BarLabelProps {
```

Comment:
in this case we are repeating the same props repeated both here and in BarList, what we should do is to extend this one in BarList so we don't repeat the same thing.

BarList.ts
```ts
export interface BarListProps extends BarLabelProps,  HTMLAttributes<HTMLDivElement> {
 // props only for BarList component
}
```

And probably you need to rename some props like `children` on the BarLabel or something like that

Reply:
I realized that it shared more props with the ListItem interface so I decided to create an interface called LabelProps and extend it in the ListItem.

The fontSizeBar prop that shares the BarLabelProps with BarListProps, I don't think it is a good idea to insert it in the Labelprops because it would be extended in the ListItem and could generate confusion.